game/screen_intro.cc

```diff
@@ -121,11 +121,11 @@ void ScreenIntro::draw_menu_options() {
 
 		// Regular option (not selected)
 		} else {
-			std::string title = opt.getName();
+			std::string title = _(opt.getName());
 			SvgTxtTheme& txt = getTextObject(title);
 			ColorTrans c(Color::alpha(opt.isActive() ? 1.0 : 0.5));
 			txt.dimensions.left(x).center(start_y + ii*0.065);
-			txt.draw(title);
+			txt.draw(_(title));
```

Comment:
erroneous double translation

Reply:
The erroneous translation is still there :P